.vscode/launch.json

```diff
@@ -15,6 +15,40 @@
       "preLaunchTask": "Debug: Desktop",
       "postDebugTask": "Stop Debug",
       "outFiles": [ "${workspaceFolder}/dist/**/*.js" ]
-    }
+    },
+    { 
```

Comment:
disregard the changes to launch.json. I am going to remove them from this PT. I made these changes so I can debug my tests as I write them